package-configuration/linux-cli/test_get_cloudify.py

```diff
@@ -144,6 +147,25 @@ def test_check_cloudify_installed_in_venv(self):
         finally:
             shutil.rmtree(tmp_venv)
 
+    def test_get_requirements_from_source_url(self):
+        installer = get_cloudify.CloudifyInstaller()
+        req_list = installer._get_default_requirement_files(cloudify_cli_url)
+        self.assertEquals(len(req_list), 1)
+        self.assertIn('dev-requirements.txt', req_list[0])
```

Comment:
why fileserver? just mock download function to pseudo download it. unit tests are supposed to be the smallest tests you can do.